store/src/java/com/zimbra/cs/mailbox/calendar/Invite.java

```diff
@@ -163,6 +163,63 @@ public class Invite {
         setRecurId(recurrenceId);
     }
 
+    Invite(MailItem.Type type, String methodStr, TimeZoneMap tzmap, CalendarItem calItem, String uid, String status,
+           String priority, String pctComplete, long completed, String freebusy, String transp, String classProp,
+           ParsedDateTime start, ParsedDateTime end, ParsedDuration duration, Recurrence.IRecurrence recurrence,
+           boolean isOrganizer, ZOrganizer org, List<ZAttendee> attendees, String name, String tag, String tagColour,
+           String loc, int flags, String partStat, Boolean rsvp, RecurId recurrenceId, long dtstamp, long lastModified,
+           int seqno, int lastFullSeqno, int mailboxId, int mailItemId, int componentNum, boolean sentByMe,
+           String description, String descHtml, String fragment, List<String> comments, List<String> categories,
+           List<String> contacts, Geo geo, String url) {
+        setItemType(type);
+        mMethod = lookupMethod(methodStr);
+        mTzMap = tzmap;
+        mCalItem = calItem;
+        mUid = uid;
+        mStatus = status;
+        mPriority = priority;
+        mPercentComplete = pctComplete;
+        mCompleted = completed;
+        mFreeBusy = freebusy;
+        mTransparency = transp;
+        mClass = classProp;
+        mClassSetByMe = sentByMe;
+        mStart = start;
+        mEnd = end;
+        mDuration = duration;
+        mIsOrganizer = isOrganizer;
+        mOrganizer = org;
+        mAttendees = attendees;
+        mName = name != null ? name : "";
+        mtag = tag;
+        mtagColour = tagColour;
+        mLocation = loc != null ? loc : "";
+        mFlags = flags;
+        mPartStat = partStat;
+        mRsvp = rsvp;
+        mSeqNo = seqno;
+        // bug 74117 : mLastSeqNo contains the sequence number for which invitation is sent to all the attendees.
+        // This will be used when accepting the replies.
+        mLastFullSeqNo = lastFullSeqno;
+        setDtStamp(dtstamp);
+        setLastModified(lastModified);
+
+        mMailboxId = mailboxId;
+        mMailItemId = mailItemId;
+        mComponentNum = componentNum;
+        mSentByMe = sentByMe;
+        setDescription(description, descHtml);
+        mFragment = fragment != null ? fragment : "";
+        mComments = comments != null ? comments : new ArrayList<String>();
+        mCategories = categories != null ? categories : new ArrayList<String>();
+        mContacts = contacts != null ? contacts : new ArrayList<String>();
+        mGeo = geo;
+        setUrl(url);
+
+        setRecurrence(recurrence);
+        setRecurId(recurrenceId);
+    }
+
```

Comment:
overload the existing constructor rather than duplicating the entire constructor code